### DISCUSSION TITLE
YouTube Popup at the End of the Video

### DISCUSSION THREAD

**Discussion by nmh1024** [uBlockOrigin/uAssets#26538] [Ideas]
Hello,

I just have a suggestion. I recently had to switch to UBlock Origin Lite since the original is no longer supported by Chrome.
One of the best features for me was the ability to turn off the annoying popups at the end of every YouTube video. It doesn't seem as though this is currently a thing on Lite, so I think it should definitely be on the list if possible. 

<img width="974" alt="Screenshot 2024-12-19 at 11 57 07 PM" src="https://github.com/user-attachments/assets/22425a5e-6fa7-4c88-928f-def0fe8bbae8" />

**Comment by krystian3w** [comment]
You can try enable "EasyList – Other Annoyances" in uBOL and test again (maybe addressed that annoying objects).

Also possible back uBO by policies:

https://github.com/uBlockOrigin/uBlock-issues/discussions/2977#discussioncomment-8134456

https://github.com/uBlockOrigin/uBlock-issues/discussions/2977#discussioncomment-9521603

Without migration to Brave/Opera/Thorium/Supremium.
